### DISCUSSION TITLE
Giving Twitter access

### DISCUSSION THREAD

**Discussion by benjaminoakes** [benjaminoakes/maid#198] [General]
@coaxial Not sure if you're on Twitter, but [Maid has an account](https://twitter.com/maidgem) that I basically haven't used in 8 years.  I'm not sure if it's still possible to share access using TweetDeck (especially given the changes recently implemented since Elon Musk's purchase of the company), but please let me know if you're interested.

**Comment by coaxial** [comment]
I don't personally use Twitter and the account doesn't seem to have many followers that would benefit from regular updates. Unless you're willing to look after the Twitter account still, I'd suggest closing it. What do you think?

**Reply by benjaminoakes** [reply]
I'll give this some thought. It might be nice to just automatically post something based on other activity that is already happening.  For example, releases or issues or commits or something like that

**Reply by coaxial** [reply]
I didn't think of that but it would be a worthwhile use of the account.

**Reply by benjaminoakes** [reply]
Yeah, as part of a larger marketing strategy, so to speak

**Reply by benjaminoakes** [reply]
That said, what really got a lot of people into using Maid in the first place was a hacker news post. Perhaps that could be replicated?

**Reply by benjaminoakes** [reply]
A blog post about open source maintainership burnout with connection to maid might be the type of thing that could do that.  It's definitely something I've felt, and adding another maintainer has been amazing so far. 🙂

So a common problem and a solution

**Reply by benjaminoakes** [reply]
I gave it a quick look.  There are a lot of RSS feeds that could post on Twitter automatically.  (I'm already personally subscribed to them and they definitely still work.)  Also, I still have access to the maidgem Twitter account and can post there.

That said, I can't use my IFTTT account to post to more than one account without upgrading and paying more, so that's not a great option.  I already use it to connect https://www.benjaminoakes.com and https://twitter.com/benjaminoakes.

We could potentially use another service to post to that account.  Perhaps Zapier.  Or maybe there's a way to use GitHub Actions for this.

However, it's not clear that this is valuable.  Posting to Twitter **could** help raise the visibility of the project, or it may not do much.  I was more interested in this idea when it was low-effort, honestly.  :smile:  That might not be the case.

Anyway, since I have them, here are some RSS feeds that I'm aware of in case we want to start republishing them someday:

https://github.com/maid/maid/releases.atom
https://github.com/maid/maid/wiki.atom
https://github.com/maid/maid/commits/master.atom
https://rubygems.org/gems/maid/versions.atom
